app/models/user.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  remember_created_at    :datetime
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
We should implement confirmable in the near future

Reply:
I thought about it, but in the end I decided to go with the same that we had on the rails_api_base

Reply:
Maybe for another PR should be good to implement confirmable module

Reply:
I think we can provide much more things out of the box here since we control both the FE and BE. It'd be cool to just plug your email provider and get everything working

I'd like to also have Google + Facebook sign in. Worst case they can be deleted.